djangocms_page_sitemap/sitemap.py

```diff
@@ -23,22 +24,32 @@ def items(self):
         # https://github.com/divio/django-cms/blob/2894ae8bcf92092d947a097499c01ab2bbb0e6df/cms/sitemaps/cms_sitemap.py
         site = get_current_site()
         languages = get_public_languages(site_id=site.pk)
+        page_content_queryset = PageContent.objects.filter(
+            page__node__site=site,
+            language__in=languages
```

Comment:
I'm thinking that we can add  `page__pagesitemapproperties__include_in_sitemap=True` here now as we are using the prefetch query.

We then no longer need the exclusion filtering below.

We would need to do a count and prove this logic.